defaults/main.yml

```diff
@@ -60,6 +61,7 @@ k8s_cluster_join_configuration:
   # the node name, the CRI socket to use or any other settings that should apply to this
   # node only (e.g. the node ip)
   nodeRegistration:
+    name: "{{ inventory_hostname }}"
```

Comment:
It's better to add separate variable like `k8s_cluster_node_name` and set respective value:
```yaml
k8s_cluster_node_name: "{{ inventory_hostname }}"
```
This variable allows you set appropriate value both for join/init config:
```yaml
k8s_cluster_join_configuration:
  nodeRegistration:
    name: "{{ k8s_cluster_node_name }}"
```
...  and node check in task:
```yaml
- name: Check nodes until ready
  ansible.builtin.shell: |
    kubectl get node {{ k8s_cluster_node_name }} \
      --kubeconfig=/etc/kubernetes/admin.conf -o json | jq -Mr '.status.conditions[4].status'
```
It prevent 'split-brain' behavior when there are different values in init/join config and node check in task